infrastructure/monitoring/promtail/README.md

```diff
@@ -0,0 +1,49 @@
+# Promtail
+
+Promtail is an agent which ships the contents of local logs to a private Grafana Loki instance. It is usually deployed to every machine that has applications needed to be monitored.
+It primarily:
+ - Discovers targets
+ - Attaches labels to log streams
+ - Pushes them to the Loki instance
+
+ ## Prerequisites
+
+Helm must be installed to use the charts value. Please follow the [Helm installing guide](https://helm.sh/docs/intro/install/) before getting starting.
+
+After Helm is ready, add the Grafana repo as follow:
+
+```helm
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+```
+To see the Grafana charts, you can run:
+
+```helm
+helm search repo grafana
+```
+
+## Installing
+
+To install the chart with the release name promtail and apply the configuration of values.yaml:
+
+```helm
+helm upgrade --install promtail -f < YOUR_PATH >/values.yaml grafana/promtail
+```
+
+The [values.yaml](./values.yaml) is the file value for Promtail configuration.
+
+```yaml
+  snippets:
+    pipelineStages:
+      #- cri: {}
+      - docker: {}
+```
+
+The **pipelineStages** is the way of extract data by parsing the log line. You can use the [Docker standard](https://grafana.com/docs/loki/latest/clients/promtail/stages/docker/), the [CRI standard](https://grafana.com/docs/loki/latest/clients/promtail/stages/cri/), a [regular expression](https://grafana.com/docs/loki/latest/clients/promtail/stages/regex/) or by parsing the log line as [JSON](https://grafana.com/docs/loki/latest/clients/promtail/stages/json/). You can only use one of these standard. 
+
+*NOTE: for JSON, you must configure **all** the expressions of your logs manually ([Using extraced data example](https://grafana.com/docs/loki/latest/clients/promtail/stages/json/#using-extracted-data)), because it does not recognize the key-value fields of the JSON, which means non-string elements like numbers, booleans or timestamp will not be assigne to those types.*
```

Comment:
```suggestion
*NOTE: for JSON, you must configure **all** the expressions of your logs manually ([Using extraced data example](https://grafana.com/docs/loki/latest/clients/promtail/stages/json/#using-extracted-data)), because it does not recognize the key-value fields of the JSON, which means non-string elements like numbers, booleans or timestamps will not be assigned to those types.*
```